extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -709,7 +717,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
     } else {
       this.setContactPopupStyle(input);
       contactEl.find('ul').html('<li data-test="no-contact-found">No Contacts Found</li>'); // xss-direct
-      if (!this.googleContactsSearchEnabled) {
+      if (!(await this.googleContactsSearchEnabled)) {
```

Comment:
Do you think we should show "Enable Google Contact Search" button when a request to get token info fails? Wouldn't that be misleading?
I suggest to use strictly `(await this.googleContactsSearchEnabled) === false` here

Reply:
I agree